src/index.ts

```diff
@@ -84,7 +93,7 @@ export class ReduxAnalyticsManager<A, S> {
 
     this.initialized = true;
 
-    return (store) => (next) => (action) => {
+    return (store) => (next: Dispatch<AnyAction>) => (action: AnyAction) => {
```

Comment:
Thanks for adding these types! I think we can add `MiddlewareAPI` as the type for store: https://github.com/reduxjs/redux/blob/master/index.d.ts#L303